R/prepare_pctr_product.R

```diff
@@ -41,7 +41,7 @@ rename_pctr_product <- function(data) {
       benchmark = "grouped_by",
       PCTR_risk_category = "risk_category",
       ep_product = "clustered",
-      isic_name = "isic_4digit_name_ecoinvent"
+      isic_4digit_name = "isic_4digit_name_ecoinvent"
```

Comment:
* In the output of `profile_emissions()` at product level the column `isic_name` is now renamed to `input_isic_4digit_name`.

Reply:
@maurolepore it's renamed to `isic_4digit_name` (Just mentioning the typo)